assignment08/js/script.js

```diff
@@ -0,0 +1,88 @@
+window.addEventListener('load', () => {
+    
+
+    //YOUR CODE SHOULD START BELOW THIS LINE
+ let  myClock = new Clock;    
+myClock.startClock();

```

Comment:
there should have been a line here that just was:

startClock();

it said not to change that line in the starter files. That line should call the function in the clock.js file. Don't create a class and use that to start. And then take all the clock definition code out of this file. That is lines 7 through 85. That is all in clock.js.